src/frontend/js/api/joanie.spec.ts

```diff
@@ -12,21 +12,24 @@ describe('api/joanie', () => {
     expect(responseBody).toBe('');
     fetchMock.restore();
   });
+
   it('buildApiUrl should build api url', () => {
     interface TestApiFilters extends ResourcesQuery {
       firstname: string;
       lastname: string;
       gender: string[];
+      is_active: boolean;
     }
     const apiFilters: TestApiFilters = {
       id: 'DUMMY_TEST_API_ID',
       firstname: 'John',
       lastname: 'Do',
       gender: ['male', 'robot'],
+      is_active: true,
     };
-    const url = buildApiUrl('test/:id/', apiFilters);
+    const url = buildApiUrl('http://example.com/test/:id/', apiFilters);
```

Comment:
It would be nice to have a test with different type like: `null`, `undefined` and `''`